packages/opentelemetry-core/src/Exporter.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Also, would you mind moving this file to a separate directory `src/internal` so that we can easily distinguish if a file only contains internal tools?